### PR TITLE
Fix grammatical errors in README

### PR DESCRIPTION
**Pull request by gaurav-singh1998**
Found grammatical errors in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -311,17 +311,17 @@ older versions of mlpack:
 If you find a bug in mlpack or have any problems, numerous routes are available
 for help.
 
-Github is used for bug tracking, and can be found at
+Github is used for bug tracking and can be found at
```
Comment (walragatver) [inline README.md]: Nice catch.

File: README.md [hunk 2]
```diff
@@ -311,17 +311,17 @@ older versions of mlpack:
...
 https://github.com/mlpack/mlpack/.
 It is easy to register an account and file a bug there, and the mlpack
-development team will try to quickly resolve your issue.
```
Comment (walragatver) [inline README.md]: I think this was grammaticaly correct. I won't mind the new change.

File: README.md [hunk 3]
```diff
@@ -311,17 +311,17 @@ older versions of mlpack:
...
+development team will try to resolve your issue quickly.
 
-In addition, mailing lists are available.  The mlpack discussion list is
```
Comment (walragatver) [inline README.md]: I think this is correct as well.

**Comment by gaurav-singh1998** [comment]
So will it be merged?

**Comment by walragatver** [comment]
Ah, not sure (I cannot merge). However if you could find more such issues and push them in this PR then definitely. You can also find other design and style issues as well.
https://github.com/mlpack/mlpack/wiki/DesignGuidelines

**Comment by gaurav-singh1998** [comment]
Well, I am new to open-source contribution so can please tell me how find this wiki page on my local machine after I have cloned the repository onto my local machine?

File: README.md (hunk 2 above)
Comment (rcurtin) [reply README.md]: Agreed, I don't see any reason to make this change.

File: README.md (hunk 3 above)
Comment (rcurtin) [reply README.md]: Yes, I don't see any reason to change `in addition` to `also`.

File: README.md [hunk 4]
```diff
@@ -311,17 +311,17 @@ older versions of mlpack:
...
+Also, mailing lists are available.  The mlpack discussion list is
 available at
 
   [mlpack discussion list](http://lists.mlpack.org/mailman/listinfo/mlpack)
 
-and the git commit list is available at
+moreover, the git commit list is available at
```
Comment (rcurtin) [inline README.md]: And I also see no reason to change `and` to `moreover`.

**Pull request closed without merging**